modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -498,15 +495,11 @@ def tracked_item_req_date(tracked_item, item)
         end
 
         def get_evss_documents(claim_id)
-          ClaimsApi::Logger.log('EVSS', rid: request.request_id, detail: 'getting docs')
-          docs = evss_docs_service.get_claim_documents(claim_id).body
-          ClaimsApi::Logger.log('EVSS', rid: request.request_id, detail: 'got docs')
-          docs
+          evss_docs_service.get_claim_documents(claim_id).body
         rescue => e
-          ClaimsApi::Logger.log('EVSS', rid: request.request_id, detail: 'getting docs failed', exception: e)
-          log_message_to_sentry('Error in Claims v2 show calling EVSS Doc Service',
-                                :warning,
-                                body: e.message)
+          claims_v2_logging('evss_doc_service',
+                            message: "getting docs failed in claims controller with e.message:
+                            #{e.message}, rid: #{request.request_id}")
```

Comment:
Extra newline here...got the following in the console when I triggered this locally:

2023-10-31 16:05:08.017141 I [48006:puma srv tp 005] Rails -- ClaimsApi :: evss_doc_service :: {"icn":"1012666182V203559","cid":"0oak7plybj1iDnw7q2p7","current_user":null,"message":"getting docs failed in claims controller with e.message:\n                            StandardError, rid: f5b02c12-e715-4f35-99ac-c239df11fd5d","api_version":"V2","poa":null}